database/mssql/scripts/dbscripts/PSP_PIMS_S25_00/Build/02_PSP_PIMS_S25_00_Build.sql

```diff
@@ -0,0 +1,19836 @@
+GO
+EXEC sp_addextendedproperty 
+	@name = N'MS_Description', @value = N'Lease/licence amount' , 
+	@level0type = N'Schema', @level0name = N'dbo', 
+	@level1type = N'Table', @level1name = N'PIMS_LEASE', 
+	@level2type = N'Column', @level2name = N'LEASE_AMOUNT'
+GO
+EXEC sp_addextendedproperty 
+	@name = N'MS_Description', @value = N'Date current responsibility came into effect for this lease' , 
+	@level0type = N'Schema', @level0name = N'dbo', 
+	@level1type = N'Table', @level1name = N'PIMS_LEASE', 
+	@level2type = N'Column', @level2name = N'RESPONSIBILITY_EFFECTIVE_DATE'
+GO
+EXEC sp_addextendedproperty 
+	@name = N'MS_Description', @value = N'Inspection date' , 
+	@level0type = N'Schema', @level0name = N'dbo', 
+	@level1type = N'Table', @level1name = N'PIMS_LEASE', 
+	@level2type = N'Column', @level2name = N'INSPECTION_DATE'
+GO
+EXEC sp_addextendedproperty 
+	@name = N'MS_Description', @value = N'Notes accompanying inspection' , 
+	@level0type = N'Schema', @level0name = N'dbo', 
+	@level1type = N'Table', @level1name = N'PIMS_LEASE', 
+	@level2type = N'Column', @level2name = N'INSPECTION_NOTES'
+GO
+EXEC sp_addextendedproperty 
+	@name = N'MS_Description', @value = N'Is subject the Residential Tenancy Act' , 
+	@level0type = N'Schema', @level0name = N'dbo', 
+	@level1type = N'Table', @level1name = N'PIMS_LEASE', 
+	@level2type = N'Column', @level2name = N'IS_SUBJECT_TO_RTA'
+GO
+EXEC sp_addextendedproperty 
+	@name = N'MS_Description', @value = N'Is a commercial building' , 
+	@level0type = N'Schema', @level0name = N'dbo', 
+	@level1type = N'Table', @level1name = N'PIMS_LEASE', 
+	@level2type = N'Column', @level2name = N'IS_COMM_BLDG'
+GO
+EXEC sp_addextendedproperty 
+	@name = N'MS_Description', @value = N'Is improvement of another description' , 
+	@level0type = N'Schema', @level0name = N'dbo', 
+	@level1type = N'Table', @level1name = N'PIMS_LEASE', 
+	@level2type = N'Column', @level2name = N'IS_OTHER_IMPROVEMENT'
+GO
+EXEC sp_addextendedproperty 
+	@name = N'MS_Description', @value = N'Incidcator that lease/license has expired' , 
+	@level0type = N'Schema', @level0name = N'dbo', 
+	@level1type = N'Table', @level1name = N'PIMS_LEASE', 
+	@level2type = N'Column', @level2name = N'IS_EXPIRED'
+GO
+EXEC sp_addextendedproperty 
+	@name = N'MS_Description', @value = N'Indicator that phyical file exists' , 
+	@level0type = N'Schema', @level0name = N'dbo', 
+	@level1type = N'Table', @level1name = N'PIMS_LEASE', 
+	@level2type = N'Column', @level2name = N'HAS_PHYSICAL_FILE'
+GO
+EXEC sp_addextendedproperty 
+	@name = N'MS_Description', @value = N'Indicator that digital file exists' , 
+	@level0type = N'Schema', @level0name = N'dbo', 
+	@level1type = N'Table', @level1name = N'PIMS_LEASE', 
+	@level2type = N'Column', @level2name = N'HAS_DIGITAL_FILE'
+GO
+EXEC sp_addextendedproperty 
+	@name = N'MS_Description', @value = N'Indicator that physical license exists' , 
+	@level0type = N'Schema', @level0name = N'dbo', 
+	@level1type = N'Table', @level
```

Comment:
Is the "NOT ENFORCED" part intended to be there? It might be removing the constraints on said tables.

Reply:
The NOT ENFORCED should not be there.  I'll recheck.

Reply:
NOT ENFORCED has been removed from the build and PR created.